storage-providers/filecoin-deals/deal-renewal-101.md

```diff
@@ -0,0 +1,188 @@
+---
+description: >-
+  This page discusses the basic 101-level knowledge of deal renewal for storage providers (and clients).
+---
+
+# Deal Renewal 101
+
+Renewing Filecoin storage deals involves a set of steps to ensure that your data remains stored on the Filecoin network as your original storage deal approaches its expiration. Here is an outline of how that process might look like.
+
+We will demonstrate most of the commands with [Lotus](../../nodes/full-nodes/basic-setup.md#basic-setup), but hopefully you can follow along using other alternatives.
+
+## 1. Monitor Deal Expirations
+
+- **Track deal expiration**: Use a tool like Lotus or Filfox (Filecoin Explorer) to monitor the expiration dates of your existing storage deals.
+- **Set up alerts**: It’s a good practice to set up notifications or alerts so you are aware when deals are nearing expiration.
+
+Here are basic ways you can track and monitor deals:
+
+### 1.1 Check Deal Expiration Using Lotus CLI
+
+Use the Lotus CLI to query information about your active storage deals:
+
+```bash
+lotus client list-deals
```

Comment:
The entire `client` subcommand no longer exists in `lotus`, see: https://github.com/filecoin-project/lotus/pull/11999. Considering that the rest of this tutorial is based around this command existing, I think we park this current iteration and figure out what the baseline deal interactions are using Boost.